libraries/AP_UAVCAN/dsdl/dronecan/remoteid/20033.System.uavcan

```diff
@@ -0,0 +1,40 @@
+#
+# DroneCAN version of MAVLink OPEN_DRONE_ID_SYSTEM
+# see MAVLink XML for detailed description
+#
+uint8[<=20] id_or_mac
+
+uint8 ODID_OPERATOR_LOCATION_TYPE_TAKEOFF = 0 # The location of the operator is the same as the take-off location
+uint8 ODID_OPERATOR_LOCATION_TYPE_LIVE_GNSS = 1 # The location of the operator is based on live GNSS data
+uint8 ODID_OPERATOR_LOCATION_TYPE_FIXED = 2 # The location of the operator is a fixed location
+uint8 operator_location_type
+
+uint8 ODID_CLASSIFICATION_TYPE_EU = 0
+uint8 classification_type
```

Comment:
@rabad-matternet @tridge this classification type field makes it seem like there is only the Classification Type EU. However in the F3411-22a block it mentions:

Classification Type: Bits [4..2]
0 = Undeclared
1 = European Union
2–7 = Reserved

Is this implementation incorrect?